app/models/katello/concerns/content_aspect_host_extensions.rb

```diff
@@ -0,0 +1,22 @@
+module Katello
+  module Concerns
+    module ContentAspectHostExtensions
+      extend ActiveSupport::Concern
+
+      included do
+        has_one :content_aspect, :class_name => '::Katello::Host::ContentAspect', :foreign_key => :host_id, :inverse_of => :host, :dependent =>  :destroy
+        has_one :content_view, :through => :content_aspect
+        has_one :lifecycle_environment, :through => :content_aspect
```

Comment:
My understanding was that the aspect isolates and contains certain properties that are specific to the aspect (and that without the aspect, those go away). So for the relationship access to exist, even as short cuts, on the main object seems to confuse that for me especially if I am reading through the code.